plugin/hosts/README.md

```diff
@@ -6,13 +6,40 @@
 
 ## Description
 
-The hosts plugin is useful for serving zones from a /etc/hosts file. It serves from a preloaded
+The hosts plugin is useful for serving zones from a `/etc/hosts` file. It serves from a preloaded
 file that exists on disk. It checks the file for changes and updates the zones accordingly. This
 plugin only supports A, AAAA, and PTR records. The hosts plugin can be used with readily
 available hosts files that block access to advertising servers.
 
 This plugin can only be used once per Server Block.
 
+## The hosts file
+
+Entries in the `/etc/hosts` file are based on IETF RFC-952 which was updated by IETF RFC-1123. Commonly the entries are of the from `IP_address canonical_hostname [aliases...]` as explained by the `hosts` man page (`man 5 hosts`).
```

Comment:
Can you also add/create the RFC to a (new?) See Also section at the end of this page?